tests/docs/expected.js

```diff
@@ -32,7 +32,6 @@ module.exports = {
     '_helpers',
     '_hydrateUnsuppliedQueryParams',
     '_initializersRan',
-    '_injections',
     '_internalReset',
     '_invoke',
     '_lazyInjections',
```

Comment:
I checked out `lazyInjections` and it looks like it is used to validate service injections and not implicit.

https://github.com/emberjs/ember.js/blob/b818f5408836867bdf725583126c994080eb0bf1/packages/%40ember/-internals/glimmer/tests/integration/components/curly-components-test.js#L2898